pobi.md

```diff
@@ -54,74 +54,73 @@ Note that the value of L1_Proof_Height is less than L1_Block_Height.
+The solution is for the enclave to serially (on a single thread) calculate a large enough number of SHA256 hashes, which it wouldn't be able to do faster than an average block time even on powerful hardware.
 
-This solution is effective, since the code is attested, and does not rely on any input from the host.
+This solution is effective since the code is attested and does not rely on any input from the host.
 
-A node operator wanting to cheat would restart the enclave, and quickly feed it the proof of inclusion, but the enclave will only process it after 15 seconds, which means the operator has already missed the time window for that rollup.
+A node operator wanting to cheat would restart the enclave and quickly feed it the proof of inclusion, only for the enclave to process it after 15 seconds, which means the operator has already missed the time window for that rollup.
 
-This built-in startup delay is also useful in preventing other real time side channel attacks, which could be used for MEV.
+This built-in startup delay is also useful in preventing other real-time side-channel attacks, which could be used for MEV.
 
 ### Aggregator Incentives
-All successful decentralised solutions need a strong incentive mechanism to keep the protocol functioning effectively.
+All successful decentralised solutions need a robust incentive mechanism to keep the protocol functioning effectively.
 
-The Bitcoin incentive model is very simple. Each transaction pays a fee and on top of that each block contains a coinbase transaction, both going to the winner. If there are re-organisations of the chain, the block that makes it onto the canonical chain is the one that pays the reward, because it is in the ledger. This mechanism provides the right incentives for miners to follow the rules. One disadvantage of this model is that fees can get very high in periods of network congestion, which degrades user experience.
+The Bitcoin incentive model is simple. Each transaction pays a fee and on top of that each block contains a coinbase transaction, both going to the winner. If there are re-organisations of the chain, the block that makes it onto the canonical chain is the one that pays the reward because it is in the ledger. This mechanism provides the right incentives for miners to follow the rules. One disadvantage of this model is that fees can get very high in periods of network congestion, which degrades the user experience.
 
-Ethereum builds on top of this model to handle the complexities of Turing complete smart contract execution, by introducing a notion of _gas_ and of _gas price_, but the high level mechanics remain the same.
+Ethereum builds on top of this model to handle the complexities of Turing complete smart contract execution by introducing a notion of _gas_ and of _gas price_, but the high-level mechanics remain the same.
```

Comment:
Does "Turing complete" need to be added to the Glossary? If so please add to this PR.